stylesheets/pages/quiz.css

```diff
@@ -0,0 +1,53 @@
+@import url(../base.css);
+/* Quiz Page  */
+
+.quiz-question-wrapper{
+    width: 90%;
+    max-width: 500px;
```

Comment:
You can use rem here instead of px

Reply:
Thank you for pointing it out, I have fixed it in the latest commit